docs/source/user/debugger.rst

```diff
@@ -12,11 +12,17 @@ Requirements
 
 For the debugger to be enabled and visible, a kernel with support for debugging is required.
 
-For now ``xeus-python`` is the only Jupyter Kernels for Python that supports debugging.
-However other Jupyter Kernels can also support debugging and be compatible with the JupyterLab debugger
+Here is a list of kernels that are known to be supporting the debug protocol:
+
+- `xeus-python <https://github.com/jupyter-xeus/xeus-python>`_: Jupyter kernel for the Python programming language
+- `xeus-robot <https://github.com/jupyter-xeus/xeus-robot>`_: Jupyter kernel for Robot Framework
+
+Other Jupyter Kernels can also support debugging and be compatible with the JupyterLab debugger
 by implementing the `Jupyter Debugger Protocol <https://jupyter-client.readthedocs.io/en/latest/messaging.html#debug-request>`_.
 
-To install ``xeus-python`` in a new ``conda`` environment:
+If you know of other kernels with support for debugging, please open a PR to add them to this list!
```

Comment:
```suggestion
If you know of other kernels with support for debugging, please open a PR to add them to this list.
```

I know you are excited about getting new kernels. To me this seems a bit much for documentation :).

Reply:
Haha yes I think it was written with some kind of "blog post tone" to it, but it makes sense to follow the style of the documentation :)

Thanks!